### PR TITLE
[j4.0] Readme update for J4

### PR DESCRIPTION
**Pull request by brianteeman**
Updated links etc for J4

@MATsxm I assume the wiki links are ok. Not sure what the process is for creating them. ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,12 +68,13 @@ Where can you get support and help?
...
 ---------------------
-* What's [new in Joomla! 3.x](https://www.joomla.org/3)?
-* What are the [main differences between 2.5 and 3.x](https://docs.joomla.org/What_are_the_major_differences_between_Joomla!_2.5_and_3.x%3F)?
-* How to [migrate from 2.5.x to 3.x](https://docs.joomla.org/Joomla_2.5_to_3.x_Step_by_Step_Migration).
-* How to [migrate from 1.5.x to 3.x](https://docs.joomla.org/Joomla_1.5_to_3.x_Step_by_Step_Migration).
+* What's [new in Joomla! 4.x](https://www.joomla.org/4)?
+* What are the [main differences between 3.x and 4.x](https://docs.joomla.org/What_are_the_major_differences_between_Joomla!_3.x_and_4.x)?
+* How to [migrate from 3.x to 4.x](https://docs.joomla.org/Joomla_3.x_to_4.x_Step_by_Step_Migration).
+* How to [migrate from 2.5.x to 4.x](https://docs.joomla.org/Joomla_2.5_to_3.x_Step_by_Step_Migration).
+* How to [migrate from 1.5.x to 4.x](https://docs.joomla.org/Joomla_1.5_to_3.x_Step_by_Step_Migration).
```
Comment (wilsonge) [inline README.md]: Let's leave this as 1.5.x to 3.x and 2.5.x to 3.x. Because they are going to have to go 1.5 to 2.5, 2.5 to 3.x then 3.x to 4.0

**Comment by wilsonge** [comment]
👍

File: README.txt [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+	* This is a Joomla! 4.x installation/upgrade package.
 	* Joomla! Official site: https://www.joomla.org
 	* Joomla! 4.0 version history - https://docs.joomla.org/Joomla_4.0_version_history
 	* Detailed changes in the Changelog: https://github.com/joomla/joomla-cms/commits/master
 
 2- What is Joomla?
 	* Joomla! is a Content Management System (CMS) which enables you to build Web sites and powerful online applications.
 	* It's a free and OpenSource software, distributed under the GNU General Public License version 2 or later.
-	* This is a simple and powerful web server application and it requires a server with PHP and either MySQL, PostgreSQL, or SQL Server to run.
+	* This is a simple and powerful web server application and it requires a server with PHP and either MySQLor  PostgreSQL to run.
```
Comment (Hils) [inline README.txt]: should it be 'MySQL or' and not 'MySQLor '? line 10

File: README.txt [hunk 3]
```diff
@@ -52,9 +52,10 @@
 	* Post questions at our forums: http://forum.joomla.org
 	* Joomla! Resources Directory (JRD): http://resources.joomla.org/
 
-11- Do you already have a Joomla! site that's not built with Joomla! 3.x ?
-	* What's new in Joomla! 3.x: https://www.joomla.org/3
-	* What are the main differences from 2.5 to 3? https://docs.joomla.org/What_are_the_major_differences_between_Joomla!_2.5_and_3.x%3F
+11- Do you already have a Joomla! site that's not built with Joomla! 4.x ?
+	* What's new in Joomla! 4.x: https://www.joomla.org/4
+	* What are the main differences from 3.x to 4.x? https://docs.joomla.org/What_are_the_major_differences_between_Joomla!_3.x_and_4.x
```
Comment (Hils) [inline README.txt]: main differences 'between' rather than 'from' line 57

Comment (brianteeman) [inline README.txt]: Thanks - fixed

File: README.txt (hunk 2 above)
Comment (brianteeman) [inline README.txt]: thanks fixed

File: README.md (hunk 1 above)
Comment (brianteeman) [inline README.md]: Thanks - i kept it in the txt it was a mistake in the md. fixed

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**